draft-ietf-quic-qlog-quic-events.md

```diff
@@ -918,27 +916,28 @@ TransportDatagramsReceived = {
     ; to support passing multiple at once
     ? count: uint16
 
-    ; RawInfo:length field indicates total length of the datagrams
-    ; including UDP header length
+    ; The RawInfo fields do not include the UDP headers, only the UDP payload
     ? raw: [+ RawInfo]
 
     ? datagram_ids: [+ uint32]
 }
 ~~~
 {: #transport-datagramsreceived-def title="TransportDatagramsReceived definition"}
 
-Note: for more details on "datagram_ids", see {{transport-datagramssent}}.
+For more details on "datagram_ids", see {{transport-datagramssent}}.
 
 ## datagram_dropped {#transport-datagramdropped}
 Importance: Extra
 
 When a UDP-level datagram is dropped. This is typically done if it does not
-contain a valid QUIC packet (in that case, use packet_dropped instead).
+contain a valid QUIC packet. If it does, but the QUIC packet is dropped for
+other reasons, {{transport-packetdropped}} should be used instead.
```

Comment:
This would just render as a section number right? That's not very reader-friendly.
```suggestion
other reasons, packet_dropped ({{transport-packetdropped}}) should be used instead.
```